_posts/2020-09-29-becoming-mobile-first-at-artsy.markdown

```diff
@@ -0,0 +1,340 @@
+peer learning groups.
+
+## Aligning Product Releases
+
+Now that engineers had a solid grasp of how to build software in our application, we could re-align our product
+development process around iOS. I can't speak to the design side of this, but from a product perspective the most
+critical milestone was defining a regular 2-week app release cadence. Let me explain.
+
+Prior to the Mobile Experience team, we released the app pretty irregularly. We would release whenever we had
+something big to release, basically. There are two major flaws with that approach. The first problem is that since
+each release was bigger, each release was scarier. No one really felt confident releasing app updates. The second
+problem was that large pieces of work tended to get coupled together. This came to a head last summer when we were
+blocked from releasing an redesigned artwork view because we were waiting for a major overhaul to Artsy's GraphQL
+API to be completed. Without guidance or structure, different teams were building big projects and both had their
+changes in our default branch – it was a bit chaotic.
+
+These two problems are incidental to how we worked at Artsy but there is another, inherent problem to developing
+mobile apps:
+[deploying iOS software is weird](https://artsy.github.io/blog/2020/03/02/ios-deploys-super-weird-totally-normal/).
+Engineers, designers, and product managers at Artsy are used to being able to quickly and cheaply deploy software
+to the _web_, not the App Store. iOS software is deployed to our user's hardware, not to servers we control, which
+introduces the possibility that users might no upgrade. Software we shipped years ago is still being ran today – we
+have the anayltics to prove it. Not to mention that every app update has to go through Apple's App Store review
+process. Getting our product team aligned on a release schedule might also help us get aligned on the weirdness of
+deploying iOS software.
+
+<center>
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">iOS developers! I have a question for you. I hear a lot about teams releasing app updates on a 2-week cadence, to increase user confidence/App Store ranking/team morale/etc.<br><br>Have any teams shared their experiences with this? Bonus points for any quantitative data. Thanks!! 🤗</p>&mdash; Ash Furrow (@ashfurrow) <a href="https://twitter.com/ashfurrow/status/1184845989510811649?ref_src=twsrc%5Etfw">October 17, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</center><br />
+
+As the Mobile Experience team formed, I reached out to other mobile teams to find out about how they structured
+regular releases. [Matt Greenwell](https://twitter.com/Cform) from BuzzFeed was really helpful in particular,
+outlining their experience of the pros and cons of a regular release cadence. We implemented a two-week release
```

Comment:
Really nice to see you mention your process to move to two-week release cadence.